roman.py

```diff
@@ -1,13 +1,17 @@
+import re
+
 def roman_to_integer(value):
     number_dict = {'I':1, "V":5, "X":10, "L":50}
     total = 0
     i_sum = value.count("I")
```

Comment:
Are we using these variables? 
`i_sum , v_sum `

Reply:
Good finding!. I am not using them on this PR, I should have removed but forgot.